infra/resources/modules/commons/function_consumptions.tf

```diff
@@ -252,7 +252,8 @@ module "function_consumptions_autoscaler" {
       statistic_decrease        = "Max"
       time_aggregation_increase = "Maximum"
       time_aggregation_decrease = "Maximum"
-    }
+    },
+    memory = null
```

Comment:
**note**: Memory is not a relevant metric for autoscaling the consumer function.